packages/binding.core/src/textInput.ts

```diff
@@ -143,20 +153,28 @@ class TextInputIE extends TextInput {
   // element was changed.
   selectionChangeHandler (event) {
       const target = this.activeElement
-      const handler = target && domData.get(target, selectionChangeHandlerName)
+      
+      const handler = target && domData.get(target, 'selectionChangeHandlerName')
       if (handler) { handler(event) }
   }
 
-  watchForSelectionChangeEvent (element, ieUpdateModel) {
+  
+  // All variables are not found!
+  watchForSelectionChangeEvent (element?) {
```

Comment:
it's unclear what's happening here and it doesn't seem related to the change of types so we might consider a separate PR to make sure the implications are well understood.

Could we break this particular change into a separate PR?

Reply:
At this point we noticed through the typing that the old IE 8/9 logic could not have worked at all. We made it tsc compliant for now and added a FIXME maker.

I think that we can in a later separate pr, remove all IE8 to IE10-Logic. IE11 could be discussed.